tools/generate_md_docs.py

```
@@ -313,15 +312,20 @@ def _resolve_ollama_host() -> str:
+        # 2. Auto-detect WSL — use default gateway as Windows host IP
         try:
             with open('/proc/version', 'r') as f:
                 if 'microsoft' in f.read().lower():
-                    print("is windows")
-                    # WSL detected — read Windows host IP from resolv.conf
-                    import socket
-                    return f"http://{socket.gethostname()}.local:11434"
-        except (FileNotFoundError, IndexError, PermissionError):
+                    import subprocess
+                    result = subprocess.run(
+                        ['ip', 'route', 'show', 'default'],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)